### PR TITLE
system default nameserver in windows

### PR DESCRIPTION
**Pull request by huangnauh**
```
 .\doggo.exe github.com
NAME            TYPE    CLASS   TTL     ADDRESS         NAMESERVER
github.com.     A       IN      43s     140.82.114.4    208.67.222.222:53
github.com.     A       IN      43s     140.82.114.4    208.67.220.220:53
```

### CODE REVIEW

**Comment by mr-karan** [comment]
@huangnauh Thanks a lot. I'll review the changes soon :+1:

File: pkg/config/config_windows.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			}
+			dnsServers = append(dnsServers, ip.String())
+		}
+	}
+	return dnsServers, nil
+}
+
+// GetDefaultServers get system default nameserver
+func GetDefaultServers() ([]string, int, []string, error) {
+	// TODO: DNS Suffix
```
Comment (mr-karan) [inline pkg/config/config_windows.go:117]: @huangnauh what's this TODO about?

Comment (huangnauh) [reply pkg/config/config_windows.go:117]: https://docs.microsoft.com/en-us/windows/win32/api/iptypes/ns-iptypes-ip_adapter_dns_suffix

Similar to the search list defined in resolv.conf

Comment (mr-karan) [reply pkg/config/config_windows.go:117]: @huangnauh Noted. I guess we can merge this without the `search list` support on Windows if it's too complicated?

Comment (huangnauh) [reply pkg/config/config_windows.go:117]: Unlike resolv.conf, each adapter may have a DNS suffix list. 
Similar on mac  http://hints.macworld.com/article.php?story=20050621051643993
I think this issue can be handled in another pr.

Comment (mr-karan) [reply pkg/config/config_windows.go:117]: Sure, that makes sense.

**Comment by mr-karan** [comment]
@huangnauh Thanks again :) Learnt about https://golang.org/cmd/go/#hdr-Build_constraints via your patch. :+1:

**Pull request merged**